### ISSUE TITLE
There is empty builder after opening the old daisy chain applet

### ISSUE THREAD

**Issue by natalia-muzyka** [ChildMindInstitute/mindlogger-admin-OLD-Vue#1750]
**Preconditions**
User is logged in on the admin panel

**Steps to reproduce**

1. Open admin
2. Edit the migrated daisy chain applet
3. Pay attention to the screen

**Actual result**
There is an empty builder after opening the old daisy chain applet

**Expected result**
There is appropriate content according to the migrated applet after opening the old daisy chain applet.
General requirement: Migrate all current linked activities into the activity flow system (order is retained, logical conditions and score dependency is removed)

**Requirements**:
https://app.zenhub.com/workspaces/mindlogger-webapp-5e11094d0c26311588da9626/issues/childmindinstitute/mindlogger-applet-builder/700

**Video**: https://www.screencast.com/t/ZbwfhSEz

**Environment:**
https://admin-staging.mindlogger.org/
Win 10 / Chrome 103
ml3_general_acc@protonmail.com	/ 12345678
Ped screenr ML3, Ped Screener ML3 chain
Applet password	Qwe123!!!

**Comment by natalia-muzyka** [comment]
regarding the migration of the old reports, I see examples in design:
https://www.figma.com/file/VYuZ9vFaPfK6JnXeAXG8wW/Pre-SRC-Features?node-id=112%3A2248
@WorldImpex  was this declined and decided only to migrate the flow without migrating the reports?

**Comment by WorldImpex** [comment]
I don't think we can migrate reports directly from the original implementation so for now, we are not worried about supporting previously created reports. @natalia-muzyka

**Comment by natalia-muzyka** [comment]
Verified as fixed: https://www.screencast.com/t/bU8WnTr7

Environment:
https://admin-staging.mindlogger.org/
Win 10 / Chrome 103
ml3_general_acc@protonmail.com /12345678
Applet password Qwe123!!!

**Comment by natalia-muzyka** [comment]
The issue is reproducible in production.

Video: https://www.screencast.com/t/bJpvSPDfnay

Environment:
https://admin.mindlogger.org/
Win 10 / Chrome 105
natalia.muzyka@qatestlab.net / 19911170
switch to account: Wil
applet: RUMC - PMHSA (5-10)
